activities/tool-management/google-workspace.md

```diff
@@ -0,0 +1,22 @@
+---
+type: resource
+---
```

Comment:
Please add a redirect from the Gsuite file you removed so we don't end up a dead URL!

Reply:
Not sure which URL you mean

Reply:
You've deleted the Gsuite file, which was probably indexed by a search engine. 'No link left behind' is good practice for website maintainers - basically, any URL that ever existed should have either a redirect put in place to the current version of the content, or (exceptionally) a message put up explaining why the content was removed. 
Here's an example of an About redirect: https://github.com/publiccodenet/about/blob/develop/activities/codebase-stewardship-assessment/assessment-issue-template.md

Reply:
It can be removed from the search engine with the Google sys admin removal tool. But, where in the PR / index does the redirect go? I think it's probably faster if you do it so I see exactly where it was suppose to go. Then I learned it by seeing it and we save time (already wasted 25 min on this now) 👍🏼